.github/workflows/test.yml

```
@@ -15,6 +15,9 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Install
-        run: yarn install
+        run: |
+          yarn install
+          yarn add textlint-scripts --dev
+          yarn upgrade
```

I'm sorry if I've misled you.

```suggestion
        run: yarn install
```

It is not needed to change because `package.json` and `yarn.lock` already includes `textlint-scripts`.